helm/amko/crds/gslbhostrule_def.yaml

```diff
@@ -129,6 +129,9 @@ spec:
                     ip:
                       description: "Public IP of the corresponding site"
                       type: string
+                  required:
```

Comment:
we must take care of this in the `pkg/apis/v1alpha1/types.go`. The `omitempty` is not required if the field is required.

Reply:
removed